### PR TITLE
feat(bemtree): tech now supports engineOptions

### PR DESCRIPTION
**Pull request by JiLiZART**

### CODE REVIEW

**Comment by blond** [comment]
add docs please

**Comment by miripiruni** [comment]
…and tests [like this](https://github.com/enb/enb-bemxjst/commit/f9f05796ef3a6ca387f71e9e113d6e14fb7f4953).

File: api.ru.md [hunk 1]
```diff
@@ -181,6 +181,7 @@ bemtree
 * [requires](#requires)
 * [exportName](#exportname)
 * [forceBaseTemplates](#forceBaseTemplates)
+* [engineOptions](#engineoptions)
```
Comment (blond) [inline api.ru.md]: `#engineoptions` -> `#engineoptions-1`

**Comment by blond** [comment]
Included in [v8.3.0](https://github.com/enb/enb-bemxjst/releases/tag/v8.3.0).

**Pull request merged**